packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -33,11 +33,17 @@ export interface ChatCompositeProps extends BaseCompositeProps<ChatCompositeIcon
    * A callback for customizing the typing indicator renderer.
    */
   onRenderTypingIndicator?: (typingUsers: CommunicationParticipant[]) => JSX.Element;
-
   /**
    * Flags to enable/disable visual elements of the {@link ChatComposite}.
    */
   options?: ChatCompositeOptions;
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Optimizes the composite form factor for either desktop or mobile.
+   * @remarks `mobile` is currently only optimized for Portrait mode on mobile devices and does not support landscape.
+   * @defaultValue 'desktop'
+   */
+  formFactor?: 'desktop' | 'mobile';
```

Comment:
when this goes stable we'll have to somehow remember to update storybook docs...

Reply:
I have made a note.